server/bcoinRouter.js

```diff
@@ -17,7 +17,15 @@ const routerWithClient = client => {
       );
       logger.debug('query:', query);
       logger.debug('body:', body);
+
+      // proxy the token
```

Comment:
I'm not sure how much I like this solution, but it seemed like a very simple way to handle proxying the token from the front end

Reply:
does this not automatically get passed through in the payload? I'm actually not sure how the bcoin server would handle this

Reply:
https://github.com/bcoin-org/bcurl/blob/master/lib/client.js#L172

Reply:
The path is assigned here - https://github.com/bcoin-org/bcurl/blob/master/lib/client.js#L34
and is then overwritten above, so no matter what you put as the path, it will be the hardcoded `/multisig`